crt_portal/cts_forms/templates/forms/snippets/sortable_table_heading.html

```diff
@@ -0,0 +1,18 @@
+{% load static %}
+
+
+<th scope="col" class="{% if sort_url %}sort-cell{% endif %}">
+  {% if sort_url %}
+    <a
+      href="{{sort_url}}"
+      class="sort-link display-block {% if is_descending %}sort-up{% else %}sort-down{% endif %}"
+      aria-label="Sort complaints by property {{ heading }}"
+    >
+      <span>{{ heading }}</span>
```

Comment:
You could also put some margin on this to prevent the heading from being the full width of the `th`, so there's always room for the arrow. Something like:
```
.sort-link span {
  margin-right: [width of sort arrow plus a bit of space on the left]
}